device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1521,7 +1499,6 @@ private void updateStatus(IotHubConnectionStatus newConnectionStatus, IotHubConn
 
             //invoke connection status callbacks
             log.debug("Invoking connection status callbacks with new status details");
```

Comment:
is this debug still needed?

Reply:
I guess it is since we do eventually invoke connection status callbacks

Reply:
We still have a connection status callback. We used to have 2 where 1 was deprecated. The deprecated one was removed but we are still invoking the callback on the non-deprecated connection status callback